src/fstapp_mnesia_handler.erl

```diff
@@ -0,0 +1,57 @@
+-module(fstapp_mnesia_handler).
+
+-behaviour(fstapp_handler).
+
+-export([handle_init/0 ,handle_data/1]).
+
+-include_lib("stdlib/include/qlc.hrl"). 
+
+-record(metrics, {ostype,
+		  cpu,
+		  disk,
+		  proc,
+		  total_memory,
+		  free_memory,
+		  system_total_memory,
+		  buffered_memory,
+		  cached_memory,
+		  total_swap,
+		  free_swap}).
+
+handle_init() ->
+	_ =  mnesia:create_schema([node()]),
+	_ = mnesia:create_table(metrics,[{attributes, record_info(fields, metrics)}]),
+	ok.
+
+handle_data(Metrics) ->
+	insert_data(Metrics),
+	ok.
+
+insert_data(Metrics) ->
+	{ostype, OsType} = lists:keyfind(ostype, 1, Metrics),
+	{cpu, Cpu} = lists:keyfind(cpu, 1, Metrics),
+	{disk, Disk} = lists:keyfind(disk, 1, Metrics),
+	{proc, Proc} = lists:keyfind(proc, 1, Metrics),
+	{total_memory, TotalM} = lists:keyfind(total_memory, 1, Metrics),
+	{free_memory, FreeM} = lists:keyfind(free_memory, 1, Metrics),
+	{system_total_memory, SystemTM} = lists:keyfind(system_total_memory, 1, Metrics),
+	{buffered_memory, BufferedM} = lists:keyfind(buffered_memory, 1, Metrics),
```

Comment:
do not get the buffered_memory metric, it seems it is related with linux so it is not being retrieved on mac